tests/cases/compiler/es6ExportClauseWithAssignmentInEs5.ts

```diff
@@ -0,0 +1,11 @@
+// @target: es5
+// @module: commonjs
+
+// @filename: server.ts
+var foo = 2;
```

Comment:
Oh, missed your first comment.

Hmm postfix operators are quite tricky since they return the previous value. Need to handle it separately